mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/http/TelemetryClient.java

```diff
@@ -219,7 +228,7 @@ private void sendEventsWrapped(Vector<Hashtable<String, Object>> events, Callbac
     String url = getEventsEndpoint() + "/events/v2?access_token=" + getAccessToken();
 
     // Extra debug in staging mode
```

Comment:
This comment is now lying. This is an example of why I'm not a big fan of comments :trollface: